pkg/resource/resource_test.go

```diff
@@ -638,3 +649,241 @@ func TestGetExternalTags(t *testing.T) {
 		})
 	}
 }
+
+// single test case => not using tables
+func Test_errNotControllable_NotControllable(t *testing.T) {
```

Comment:
Very late commentary here, but in future I'd like to avoid these `_` separated long test names. I prefer shorter less descriptive names that include a `reason` string that is printed if the test fails with more context.

Reply:
Hi @negz,
Would you like me to open a PR changing these test names?

Reply:
Yes please! It's not urgent at all, but I would prefer to avoid the convention creeping in. :)